extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/AuthRuntimeConfig.java

```diff
@@ -24,6 +25,16 @@ public class AuthRuntimeConfig {
     @ConfigItem(name = "policy")
     public Map<String, PolicyConfig> rolePolicy;
 
+    /**
+     * Properties file containing the client certificate common name (CN) to role mappings.
+     * Use it only if the mTLS authentication mechanism is enabled with either
+     * `quarkus.http.ssl.client-auth=required` or `quarkus.http.ssl.client-auth=request`.
+     * <p/>
+     * Properties file is expected to have the `CN=role1,role,...,roleN` format.
```

Comment:
Pursuant to my subsequent comment:

```suggestion
     * Properties file is expected to have the `CN=role1,role,...,roleN` format and should be encoded using UTF-8.
```